### PR TITLE
Issue317

### LINKED ISSUE #317: List identities and services from the command prompt using ziti-tunnel
Fetch identities and services data from the command prompt using ziti-tunnel.exe file

### PR DESCRIPTION
**Pull request by mary-dcouto**
closes #317

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Release 1.8.2
+
+## What's New
+* [#317](https://github.com/openziti/desktop-edge-win/issues/317) command line list function to fetch identities and services
```
Comment (dovholuknf) [inline release-notes.md:4]: is there any doc about the usage that could be added to the release notes or a markdown? or can someone just running it expect to get back reasonable help?

**Comment by dovholuknf** [comment]
i'm gonna just format every file in the repo after this merge. seems like i don't have the IDE set to do that - and it should...

File: service/ziti-tunnel/cli/cmdlineClientFunctions.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
+package cli
```
Comment (dovholuknf) [inline service/ziti-tunnel/cli/cmdlineClientFunctions.go:1]: new files need the copyright header pls

File: service/ziti-tunnel/cli/cmdlineParseFunctions.go [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+	"fmt"
+	"strings"
+	"text/template"
+
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/dto"
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/service"
+)
+
+func convertToIdentityCli(id *dto.Identity) dto.IdentityCli {
+	return dto.IdentityCli{
```
Comment (dovholuknf) [inline service/ziti-tunnel/cli/cmdlineParseFunctions.go:15]: What did you need this?  the 'normal' Identity struct has all these fields already (and some extra)?

File: service/ziti-tunnel/cli/cmdlineParseFunctions.go [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+		Name:        id.Name,
+		FingerPrint: id.FingerPrint,
+		Active:      id.Active,
+		Config:      id.Config.ZtAPI,
+		Status:      id.Status,
+	}
+}
+
+func convertToServiceCli(svc dto.Service) dto.ServiceCli {
+	return dto.ServiceCli{
```
Comment (dovholuknf) [inline service/ziti-tunnel/cli/cmdlineParseFunctions.go:25]: same question. seems like the Service struct already there services this?

File: service/ziti-tunnel/cmd/identities.go [hunk 5]
```diff
@@ -0,0 +1,50 @@
+package cmd
+
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
```
Comment (dovholuknf) [inline service/ziti-tunnel/cmd/identities.go:4]: "NAME HERE" and "<EMAIL ADDRESS>" :) most of the files have the header to copy. Hopefully those don't have "<EMAIL ADDRESS>"?

File: service/ziti-tunnel/cmd/services.go [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+import (
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/cli"
+	"github.com/spf13/cobra"
+)
+
+// servicesCmd represents the services command
+var servicesCmd = &cobra.Command{
+	Use:   "services",
+	Short: "Lists services from ziti-tunnel",
+	Long: `A longer description that spans multiple lines and likely contains examples
```
Comment (dovholuknf) [inline service/ziti-tunnel/cmd/services.go:28]: Long description needs to be updated

File: service/ziti-tunnel/cli/cmdlineConstants.go [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+import (
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/dto"
+	"github.com/openziti/desktop-edge-win/service/ziti-tunnel/util/logging"
+)
+
+var GET_STATUS = dto.CommandMsg{
+	Function: "Status",
+}
+
+var templateIdentity = "{{printf \"%40s\" \"Name\"}} | {{printf \"%41s\" \"FingerPrint\"}} | {{printf \"%6s\" \"Active\"}} | {{printf \"%30s\" \"Config\"}} | {{\"Status\"}}\n" +
```
Comment (dovholuknf) [inline service/ziti-tunnel/cli/cmdlineConstants.go:12]: would you be able to use the literal string here and make the escaped quotes easier to read?

File: release-notes.md (hunk 1 above)
Comment (mary-dcouto) [reply release-notes.md:4]: ziti-tunnel list -h will display the help message

**Review by dovholuknf — APPROVED** [review]

**Pull request merged**